Src/AutoFixture/EmailAddressLocalPart.cs

```diff
@@ -6,11 +6,19 @@
 
 namespace Ploeh.AutoFixture
 {
+#if NETFULL
     /// <summary>
     /// Represents the local part of the email address, defined as everything up to, but not including, the @ sign.  
     /// Since EmailAddressLocalPart is used in constructing MailAddress, enforcement of rules on a valid email address
     /// is performed by <see cref="System.Net.Mail.MailAddress"/> and not EmailAddressLocalPart other than as noted. 
     /// </summary>
+#else
+    /// <summary>
+    /// Represents the local part of the email address, defined as everything up to, but not including, the @ sign.  
+    /// Since EmailAddressLocalPart is used in constructing MailAddress, enforcement of rules on a valid email address
+    /// is performed by System.Net.Mail.MailAddress and not EmailAddressLocalPart other than as noted. 
```

Comment:
No, everything is fine with `cref` in .NET Standard. The issue is that `System.Net.Mail.MailAddress` class doesn't exist there and our build verification fails because of that ☺️Therefore, I made two versions of the comment - one with cref and other without.